frontend-react/yarn.lock

```diff
@@ -5804,10 +5804,10 @@ flux-standard-action@^2.1.1:
     lodash.isplainobject "^4.0.6"
     lodash.isstring "^4.0.1"
 
-follow-redirects@^1.0.0:
-  version "1.14.7"
-  resolved "https://registry.yarnpkg.com/follow-redirects/-/follow-redirects-1.14.7.tgz#2004c02eb9436eee9a21446a6477debf17e81685"
-  integrity sha512-+hbxoLbFMbRKDwohX8GkTataGqO6Jb7jGwpAlwgy2bIz25XtRm7KEzJM76R1WiNT5SwZkX4Y75SwBolkpmE7iQ==
+follow-redirects@^1.0.0, follow-redirects@^1.14.8:
```

Comment:
This change should be in main. Maybe a merge conflict?

Reply:
I'll pull main again just to be sure. I started with an updated main so not sure why it made this change when running `yarn`